geoalchemy2/__init__.py

```diff
@@ -15,9 +15,13 @@
 from . import functions  # NOQA
 from . import types  # NOQA
 
+import sqlalchemy
 from sqlalchemy import Table, event
 from sqlalchemy.sql import select, func, expression, text
 
+from packaging import version
+
+SQLALCHEMY_VERSION_BEFORE_14 = version.parse(sqlalchemy.__version__) < version.parse("1.4")
```

Comment:
```suggestion
SQLALCHEMY_VERSION_BEFORE_14 = version.parse(sqlalchemy.__version__) < version.parse("1.4")

def _format_select_args(*args):
    if _SQLALCHEMY_VERSION_BEFORE_14:
        return [args]
    else:
        return args
```
I think we could use a function like this to format args.
And then we could replace
```python
                    if SQLALCHEMY_VERSION_BEFORE_14:
                        stmt = select([getattr(func, drop_func)(*args)])
                    else:
                        stmt = select(getattr(func, drop_func)(*args))
```
 by
```python
                    stmt = select(*_format_select_args(getattr(func, drop_func)(*args)))
```